doc/sphinx/addendum/micromega.rst

```diff
@@ -245,11 +245,11 @@ proof by abstracting monomials by variables.
 As shown, such a goal is solved by ``intro x. psatz Z 2.``. The oracle returns the
 cone expression :math:`2 \times (x-1) + (\mathbf{x-1}) \times (\mathbf{x−1}) + -x^2`
 (polynomial hypotheses are printed in bold). By construction, this expression
-belongs to :math:`\mathit{Cone}({−x^2,x -1})`. Moreover, by running `ring` we
+belongs to :math:`\mathit{Cone}({−x^2,x -1})`. Moreover, by running :tacn:`ring` we
 obtain :math:`-1`. By Theorem :ref:`Psatz <psatz_thm>`, the goal is valid.
 
 .. [#] Support for `nat` and :math:`\mathbb{N}` is obtained by pre-processing the goal with
-  the `zify` tactic.
+  the :tacn:`zify` tactic.
```

Comment:
Unfortunately, it looks like `zify` is not documented which made the build of the manual fail with a fatal warning. You may fix this by adding a documentation stub for `zify`. This may be as little as:

```rst
.. tacn:: zify
   :undocumented:
```

Of course, the other solution is not to use `tacn` here.

Reply:
not competent to write document for zify
just deleted :tacn: